kymatio/frontend/base_frontend.py

```diff
@@ -0,0 +1,26 @@
+
+class ScatteringBase():
```

Comment:
Check all current Scattering*D files and make sure there aren't other functions we need to define.